event/monitoring.go

```diff
@@ -45,6 +45,31 @@ type CommandFailedEvent struct {
 }
 
 // CommandMonitor represents a monitor that is triggered for different events.
+//
+// For an example, the following code will match up started events with their
```

Comment:
Sorry if this wasn't clear from my previous comments, but I intended this to be a code example on the `CommandMonitor` type similar to the examples we added for CRUD methods in the documentation epic. See the [ListDatabaseNames](https://github.com/mongodb/mongo-go-driver/blob/213fb80b373f70dba4f9f516dc4c718abe41c76b/mongo/crud_examples_test.go#L26) example for a reference.

Reply:
done